packages/dynamodb/src/utils/chunk.ts

```diff
@@ -0,0 +1,11 @@
+export const chunk = <T>(items: T[], size: number): T[][] =>
+  items.reduce(
+    (prev, item) => {
+      if (prev[prev.length - 1].length >= size) {
+        prev.push([]);
+      }
+      prev[prev.length - 1].push(item);
+      return prev;
+    },
+    [[]]
+  );
```

Comment:
I personally only like to not overuse `reduce` and only using it when e. g. accumulating something into a single value.

We could do something like:
```suggestion
export const range = (len: number) => [...Array(len).keys()]
export const chunk = <T>(items: T[], size: number) => {
  const numChunks = range(Math.ceil(items.length / size))
  return numChunks.map(i => items.slice(i * size, (i + 1) * size))
}
```
Only a suggestion though, if you prefer, we can keep the reduce way.